packages/ramldoc-generator/src/cli.js

```diff
@@ -0,0 +1,46 @@
+const mri = require('mri');
+const { generateRaml } = require('./main');
+
+async function cli(args) {
+  const options = parseArgumentsIntoOptions(args);
+  promptForMissingOptions(options);
+
+  try {
+    await generateRaml(options);
+  } catch (e) {
+    console.error(e);
+    process.exit(1);
+  }
+}
+
+function parseArgumentsIntoOptions(rawArgs) {
+  const argv = rawArgs.slice(2);
+
+  const args = mri(argv, {
+    alias: { help: ['h'] },
+    default: { help: false, name: '', src: '', dest: './src/api-specs/' },
```

Comment:
Only default values should be defined here

```suggestion
    default: { help: false, dest: './src/api-specs/' },
```